pkg/tuned/controller.go

```diff
@@ -181,10 +199,10 @@ func newController(stopCh <-chan struct{}) (*Controller, error) {
 		kubeclient:  kubeclient,
 		listers:     listers,
 		clients:     clients,
-		tunedExit:   make(chan bool, 1),
+		tunedExit:   make(chan bool),
```

Comment:
let's record the rationale from moving to buffered to unbuffered

Reply:
FWICT there was never any need to use a buffered channel.  The buffered channel was never intentional and is likely a copy-and-paste mistake.

All I remember in this context is that it the past, we had a bug where TuneD would be agressively restarted by openshift-tuned when a machine would go down for reboot. This would delay reboot by a minute or two before systemd would kill TuneD and openshift-tuned with SIGKILL.

All that matters in my view is not to agressively restart TuneD when it gets SIGTERM or exits for any reason.  This is for two reasons:
  * Blocking machine reboots
  * IPIs